code/game/machinery/cell_charger.dm

```diff
@@ -16,7 +16,6 @@
 /obj/machinery/cell_charger/update_icon()
 	cut_overlays()
 	if(charging)
-		add_overlay(image(charging.icon, charging.icon_state))
```

Comment:
Why is this removed? You seem to have edited the overlays for it?

Reply:
i edited the charger sprite, that adds the powercell as an overlay, wich wouldnt show through the lid.